README.md

```diff
@@ -69,6 +69,20 @@ https://www.khronos.org/registry/OpenCL/
     $ python -m SimpleHTTPServer
 ```
 
+## Vectorization optimization
+
+```
+$ cmake -DENABLE_COMPLEX_X2=ON ..
+```
+Multiply complex numbers two at a time instead of one at a time. Requires AVX for double and SSE 1.0 for float.
```

Comment:
But why would this ever be turned off? Why is it not the default?

Reply:
I added a note that it's on by default, and it explains why you would turn it off. A system might not support AVX, but you might still want double precision. Alternatively, you might use single precision, but you might not want any vectorization whatsoever. I'm anticipating ASIC arrays, and the like.